### DISCUSSION TITLE
help getting this to work

### DISCUSSION THREAD

**Discussion by krutru** [selivan/https-ssl-cert-check-zabbix#33] [Q&A]
Hello,

I have Zabbix 6.2 alpha2 and can't get this to work.

I followed Zabbix integration (basic):

copied ssl_cert_check.sh to /usr/local/bin/
copied userparameters_ssl_cert_check.conf file into /etc/zabbix/zabbix_agentd.d 
imported zabbix_template.xml, assigned to host and filled macros. 
Testing this on a host with expired certificate and it is not alerting or doing anything

**Comment by selivan** [comment]
You may try debugging it:

```
sudo -u zabbix zabbix_agentd -t ssl_cert_check_json[google.com,443]
```

If this works fine - see what is wrong with your item and trigger configuration.
